docs/source/concepts/InstrumentAccessLayers.rst

```diff
@@ -1,25 +1,27 @@
-===================================================
+================================================================
+Instrument Access via SpectrumInfo, DetectorInfo, ComponentInfo
+================================================================
 
 .. contents::
   :local:
 
 Introduction
 ------------
 
-There are two new layers to access instrument information, ``SpectrumInfo`` and ``DetectorInfo``, that are going to be introduced to Mantid as part of the work towards Instrument 2.0. Eventually these classes will store commonly accessed information about spectra or detectors, namely masking, monitor flags, L1, L2, 2-theta and position, leading to improved performance and cleaner code.
+There are three new layers to access instrument information, ``SpectrumInfo``, ``DetectorInfo``, and ``ComponentInfo``, which are introduced to Mantid as part of the work towards Instrument 2.0. Eventually these classes will store all commonly accessed information about spectra and detectors, namely masking, monitor flags, L1, L2, 2-theta and position. In addition, ``ComponentInfo`` will provide the updated API to tree and shape related operations currently peformed by the instrument. These changes are leading to improved performance and cleaner code.
 
 A spectrum corresponds to (a group of) one or more detectors. Most algorithms work with spectra and thus ``SpectrumInfo`` would be used. Some algorithms work on a lower level (with individual detectors) and thus ``DetectorInfo`` would be used.
 
+The current instrument largely consists of ``Detectors`` and ``Components`` - all detectors are also components. ``DetectorInfo`` and ``ComponentInfo`` are the respective replacements for these. ``ComponentInfo`` introduces a new **component index** for access, and ``DetectorInfo`` introduces a new **detector index**, these will be discussed further below. 
+
 In many cases direct access to ``Instrument`` can be removed by using these layers. This will also help in moving to using indexes for enumeration, and only working with IDs for user-facing input.
 
 Current Status
 ##############
 
-``SpectrumInfo`` and ``DetectorInfo`` are currently implemented as wrapper classes in Mantid. Using the new interfaces everywhere now will help with the eventual rollout of Instrument 2.0. It also provides cleaner code and will help with the rollout of planned indexing changes. It is also necessary to provide the addition of time indexing, required to support scanning instruments.
+``SpectrumInfo``, ``DetectorInfo`` and ``ComponentInfo``  are currently implemented in a compatibility mode, and wrap some of the exiting instrument functionality in Mantid. However, using the new interfaces everywhere now will help with the eventual rollout of Instrument 2.0. It also provides cleaner code and will help with the rollout of planned indexing changes. It is also necessary to provide the addition of time indexing, required to support scanning instruments.
```

Comment:
@SimonHeybrock this is still used for `signedTwoTheta`, but not much apart from that and could be factored out easily since this is essentially static information.